specifyweb/frontend/js_src/lib/components/Formatters/Formatter.tsx

```diff
@@ -50,7 +51,7 @@ function ConditionalMapping({
   );
 
   const [isConditionFieldDisplayed, setIsConditionFieldDisplayed] =
-    React.useState(false);
+    useTriggerState(formatter.definition.conditionField !== undefined || false);
```

Comment:
this is equivalent and simper:

```suggestion
    useTriggerState(formatter.definition.conditionField !== undefined);
```